libs/store_helper/user/ebpf_registry_helper.cpp

```diff
@@ -31,8 +32,9 @@ _get_wstring_from_string(std::string text)
 void
 ebpf_close_registry_key(ebpf_store_key_t key)
```

Comment:
If these APIs are simply a wrapper around Zw and Reg APIs, I think ideally they should go to the cxplat repo; you may want to create an issue to track this.

Reply:
Maybe.  But cxplat is also intended to work on Linux too, so someone (separately) would have to do a Linux binding there.  That may still be the right answer, but maybe the word "registry" shouldn't be in the name of a cxplat API ("key_value_store" is probably fine)